src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -168,6 +173,11 @@ public override void VisitEvent(IEventSymbol symbol)
             AddAccessibilityIfRequired(symbol);
             AddMemberModifiersIfRequired(symbol);
 
+            if ((symbol.AddMethod ?? symbol.RemoveMethod)?.IsReadOnly == true)
```

Comment:
>(symbol.AddMethod ?? symbol.RemoveMethod)?.IsReadOnly == true [](start = 16, length = 61)

What if only one of `add` or `remove` are `readonly`?

Reply:
We don't provide a way to declare such a combination of event accessors in the language, so I'd write it off as a malformed symbol that we can't display reliably--we just need to avoid crashing.

Reply:
Consider adding a test for such a case - where the event is defined in metadata or as a custom `IEventSymbol` implementation.

---
In reply to: [273756309](https://github.com/dotnet/roslyn/pull/34876#discussion_r273756309) [](ancestors = 273756309)